app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx

```diff
@@ -1,20 +1,13 @@
 import React, { useState } from "react";
 import { Modal } from "neetoui";
-import notesApi from "apis/notes";
 
-export default function DeleteAlert({ refetch, onClose, selectedNoteIds }) {
+export default function DeleteAlert({ deleteNotes, onClose, selectedNoteIds }) {
   const [deleting, setDeleting] = useState(false);
-  const handleDelete = async () => {
-    try {
-      setDeleting(true);
-      await notesApi.destroy({ ids: selectedNoteIds });
-      onClose();
-      refetch();
-    } catch (error) {
-      logger.error(error);
-    } finally {
-      setDeleting(false);
-    }
+  const handleDelete = () => {
+    setDeleting(true);
+    deleteNotes();
+    setDeleting(false);
+    onClose();
   };
   return (
     <Modal
```

Comment:
Please replace this component with neetoUI [Alert](https://neeto-ui.netlify.app/#/alert) component.